meta-ibm/recipes-phosphor/ibm-acf/ibm-acf.bb

```diff
@@ -19,10 +19,12 @@ DEPENDS = " \
 
 do_install_append(){
     install -d ${D}/${sysconfdir}/acf
-    install -m 755 ${WORKDIR}/git/subprojects/ce-login/p10-celogin-lab-pub.der \
-                   ${D}/${sysconfdir}/acf/ibmacf-dev.key
+    install -d ${D}/${servicedir}/ibm-acf
+    install -m 755 ${S}/subprojects/ce-login/p10-celogin-lab-pub.der \
+                   ${D}/${servicedir}/ibm-acf/ibmacf-dev.key
 }
 
 
 FILES_${PN} += " ${base_libdir}/security/* \
-                 ${sysconfdir}/acf/*"
+                 ${sysconfdir}/acf/* \
```

Comment:
What files are in `${sysconfdir}/acf/*`?  I believe it should be an empty directory (ready for ACF file uploads); is that right?

Reply:
That is correct. Maybe I don't need to include that line since we're not installing anything.